reference/5.1/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md

```diff
@@ -170,17 +166,49 @@ values, see [about_Execution_Policies](about_Execution_Policies.md).
 
 #### -Command
 
-Executes the specified commands (with any parameters) as though they were typed
-at the PowerShell command prompt. After execution, PowerShell exits unless the
-**NoExit** parameter is specified. Any text after `-Command` is sent as a
-single command line to PowerShell. This is different from how `-File` handles
-parameters sent to a script.
+Executes the specified commands (and any parameters) as though they were typed
+at the PowerShell command prompt, and then exits, unless the `NoExit`
+parameter is specified.
+
+The value of **Command** can be `-`, a script block, or a string. If the value
+of **Command** is `-`, the command text is read from standard input.
+
+The **Command** parameter only accepts a script block for execution when it can
+recognize the value passed to **Command** as a **ScriptBlock** type. This is
+_only_ possible when running `pwsh` from another PowerShell host. The
```

Comment:
This is a 5.1 file, should this be pwsh?

Reply:
fixed